documents/operating.md

```diff
@@ -281,49 +284,79 @@ Invoices that are not approved by a project's Comptroller will not be compensate
 ### BIFFUD Tithe
 2% of the gross income for any BIFFUD project shall be retained by the company and considered general operating funds.  It's really the least you could do.  In addition, 0.4% of the gross income for any BIFFUD effort (corporate or project based) shall be set aside into our "Grown Up Pants" fund in order to pay local business privilege taxes.
 
-In addition to these fees, the Hive Mind should specify an additional "overhead" tax percentage to be retained by the company and added to the operating expense account.  Updating the universal or project specific overhead percentage requires a vote of the Hive Mind as described in the voting section.
+In addition to these fees, the Hive Mind should specify an additional "overhead" percentage to be retained by the company and added to the operating expense account.  Updating the universal or project specific overhead percentage requires a vote of the Hive Mind as described in the voting section.
+
+The tithe + overhead applies to money earned through sales, contract agreements, donations, and any other way that cash could be given to a project from an external source.  Project owners are welcome to contribute more than the 2% if they are willing and able.
```

Comment:
This is just explaining that tithe doesn't apply to internally applied money (e.g. if we as corporate overlords allocate $300 of our general operating to a project we don't take a cut back).

It also explains that people can give more than required from their project if they want to.

This isn't actually new language, it is just adding the concept of overhead to the previous language.